Options.py

```diff
@@ -766,7 +766,7 @@ class VerifyKeys(metaclass=FreezeValidKeys):
     value: typing.Any
 
     @classmethod
-    def verify_keys(cls, data: typing.List[str]):
+    def verify_keys(cls, data: typing.Iterable[str]):
```

Comment:
Not sure about this one, because as you said, `str` is technically is an `Iterable[str]`

Reply:
There isn't really anything that can be done about that in static checking. That's one of the weaknesses of Python's type system.

We could throw in a run-time check inside that function, but that would be wasteful, given that it's already checked before every call to that function.